include/slate/BaseMatrix.hh

```diff
@@ -1250,7 +1250,14 @@ template <typename scalar_t>
 void BaseMatrix<scalar_t>::gridinfo(
     GridOrder* order, int* nprow, int* npcol, int* myrow, int* mycol ) const
 {
-    if (nprow_ > 0) {
+    int mpi_size;
+    MPI_Comm_size(mpiComm(), &mpi_size);
```

Comment:
`MPI_Comm_size` should be wrapped in `slate_mpi_call`.

Reply:
Right, I will do that.